src/components/MyProfileActivityList/index.js

```diff
@@ -92,17 +92,28 @@ class MyProfileActivityList extends Component {
             >
               <List>
                 <ListItem style={{ paddingLeft: '0.5rem' }}>
-                  <Link
-                    className="gc360-link"
-                    to={`/activity/${Membership.SessionCode}/${Membership.ActivityCode}`}
-                    style={membershipItemStyle}
-                  >
-                    <Typography>
-                      <b>{Membership.ActivityDescription}</b>
-                    </Typography>
-                    <Typography>{Membership.SessionDescription}</Typography>
-                    <Typography>{Membership.ParticipationDescription}</Typography>
-                  </Link>
+                  {/* A link to the activity is only available if the user is online */}
+                  {this.props.network === 'online' ? (
+                    <Link
+                      className="gc360-link"
+                      to={`/activity/${Membership.SessionCode}/${Membership.ActivityCode}`}
+                      style={membershipItemStyle}
+                    >
+                      <Typography>
+                        <b>{Membership.ActivityDescription}</b>
+                      </Typography>
+                      <Typography>{Membership.SessionDescription}</Typography>
+                      <Typography>{Membership.ParticipationDescription}</Typography>
+                    </Link>
+                  ) : (
+                    <div style={membershipItemStyle}>
+                      <Typography>
+                        <b>{Membership.ActivityDescription}</b>
+                      </Typography>
+                      <Typography>{Membership.SessionDescription}</Typography>
+                      <Typography>{Membership.ParticipationDescription}</Typography>
+                    </div>
+                  )}
                 </ListItem>
```

Comment:
This here is to prevent the Involvement Activity from being a link since you cannot view activity profiles offline